crates/ruff/src/test.rs

```diff
@@ -134,7 +143,7 @@ fn test_contents(contents: &str, path: &Path, settings: &Settings) -> Vec<Messag
                         r#"Fixed source has a syntax error where the source document does not. This is a bug in one of the generated fixes:
 {syntax_errors}
 Last generated fixes:
-{fixes}
+{fixes}https://discord.gg/c9MhzV8aU5
```

Comment:
Is this change intentional?

Reply:
Nope! Sorry, must have had it in my clipboard :sweat: